services/cases-api/lambdas/createCase.js

```diff
@@ -104,3 +124,67 @@ async function parseJsonD(data) {
     throwError(400, error.message);
   }
 }
+
+async function getUser(personalNumber) {
+  const params = {
+    TableName: config.users.tableName,
+    Key: {
+      personalNumber,
+    },
+  };
+
+  const [error, dbResponse] = await to(dynamoDb.call('get', params));
+  if (!dbResponse) {
+    throwError(error.statusCode, error.message);
+  }
```

Comment:
To be consistent with how other lambdas is handling error we can write this like
`if (error) {
.....
}`

:)

Reply:
Makes sense, fixed now